Dockerfile

```diff
@@ -13,6 +13,7 @@ RUN a2enmod rewrite
 
 # install the PHP extensions we need
 RUN apt-get update \
+        && apt-get -y --no-install-recommends dist-upgrade \
```

Comment:
May I ask why did you add this line?

Reply:
I added this to ensure other system packages are up-to-date, since many base images have a slower release cadence that may leave unapplied security updates.

Reply:
Better keep the docker image as it comes by default from the Registry. Can you please remove this line?

Reply:
Removed at maintainer's request.